source/common/protobuf/utility.h

```diff
@@ -158,7 +160,19 @@ class MessageUtil {
     return HashUtil::xxHash64(text);
   }
 
+  static ProtoUnknownFieldsMode proto_unknown_fields;
```

Comment:
Nit: For readability, can you do an explicit initialization to Strict here.

Reply:
Can you give me an example how to initialize a non-const static in the header?
I have explicit initialization in the .cc file.

Reply:
Ah, missed that, LGTM, thanks.